modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
I'm not sure why you're removing extra newlines. They help improve readability sometimes, and this one in particular is necessary for the docs to build.